packages/@lwc/engine-core/src/framework/api.ts

```diff
@@ -84,8 +84,9 @@ function ssf(slotName: unknown, factory: (value: any, key: any) => VFragment): V
 }
 
 // [st]atic node
-function st(fragment: Element, key: Key, parts?: VStaticPart[]): VStatic {
+function st(fragmentFactory: () => Element, key: Key, parts?: VStaticPart[]): VStatic {
```

Comment:
This change is needed in the second PR but causes several changes to the template compiler fixtures, see the test fixtures for details.